doc/quickstart.rst

```diff
@@ -81,14 +81,22 @@ This method will load the complete trajectory into memory and make positions, or
 .. code-block:: python
 
     traj.load_arrays()
-    traj.N              # M  -- frame sizes
-    traj.positions      # MxNx3 array
-    traj.orientations   # MxNx4 array
-    traj.types          # MxN array
-    traj.type_ids       # MxN array
-    traj.type           # list of type names ordered by type_id
-
-    # where M=len(traj), N=max((len(f) for f in traj))
+    traj.N               # M
+    traj.positions       # MxNx3
+    traj.orientations    # MxNx4
+    traj.velocities      # MxNx3
+    traj.mass            # MxN
+    traj.charge          # MxN
+    traj.diameter        # MxN
+    traj.moment_inertia  # MxNx3
+    traj.angmom          # MxNx4
+    traj.image           # MxNx4
+    traj.types           # MxN
+    traj.type_ids        # MxN array
```

Comment:
Is `array` specified here for a reason? I would remove it here or add it everywhere, whatever makes sense.

Reply:
Sneaky array....